requirements.txt

```diff
@@ -4,3 +4,4 @@ scipy>=1.0.0
 tensorflow==1.3
 tensorflow-tensorboard>=0.1.8
 quantum-blackbird
+python-dateutil==2.8.0
```

Comment:
Don't forget to add this to `setup.py` as well! (without the version number pinning)

Reply:
Done. I used "greater than or equal to" for the version spec.